website/blog/2022-12-21-databend-weekly-73.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'This Week in Databend #73'
+date: 2022-12-21
+slug: 2022-12-21-databend-weekly
+tags: [databend, weekly]
+authors:
+- name: PsiACE
+  url: https://github.com/psiace
+  image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a powerful cloud data warehouse. Built for elasticity and efficiency. Free and open. Also available in the cloud: <https://app.databend.com> .
+
+## What's New
+
+Check out what we've done this week to make Databend even better for you.
+
+### Features & Improvements :sparkles:
+
+**Multiple Catalogs**
+
+- implement show tables (from|in catalog.database) ([#9153](https://github.com/datafuselabs/databend/pull/9153))
+
+**Planner**
+
+- introduce histogram in column statistics ([#9310](https://github.com/datafuselabs/databend/pull/9310))
+
+**Query**
+
+- support attaching stage for insert values ([#9249](https://github.com/datafuselabs/databend/pull/9249))
+- add native format in fuse table ([#9279](https://github.com/datafuselabs/databend/pull/9279))
+- add internal_enable_sandbox_tenant config and sandbox_tenant ([#9277](https://github.com/datafuselabs/databend/pull/9277))
+
+**Sqllogictest**
+
+- introduce rust native sqllogictest framework ([#9150](https://github.com/datafuselabs/databend/pull/9150))
+
+### Code Refactoring :tada:
+
+**\***
+
+- unify apply_file_format_options for copy & insert ([#9323](https://github.com/datafuselabs/databend/pull/9323))
+
+**IO**
+
+- remove unused code ([#9266](https://github.com/datafuselabs/databend/pull/9266))
+
+**meta**
+
+- test watcher count ([#9324](https://github.com/datafuselabs/databend/pull/9324))
+
+**Planner**
+
+- replace TableContext in planner with PlannerContext ([#9290](https://github.com/datafuselabs/databend/pull/9290))
+
+### Bug Fixes :wrench:
+
+**Base**
+
+- try fix SIGABRT when catch unwind ([#9269](https://github.com/datafuselabs/databend/pull/9269))
+- replace #[thread_local] to thread_local macro ([#9280](https://github.com/datafuselabs/databend/pull/9280))
+
+**Query**
+
+- fix unknown database in query without relation to this database ([#9250](https://github.com/datafuselabs/databend/pull/9250))
+- fix wrong current_role when drop the role ([#9276](https://github.com/datafuselabs/databend/pull/9276))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Introduce Rust Native Sqllogictest Framework
+
+Sqllogictest is a program designed to verify that an SQL database engine computes correct results by comparing the results to identical queries from other SQL database engines. 
+
+In the past, Databend has run this test using a program written in Python and has migrated a large number of test cases from other successful databases. Recently, we have reimplemented this program with [sqllogictest-rs](https://github.com/risinglightdb/sqllogictest-rs) using Rust.
```

Comment:
In the past, Databend ran such tests using a program written in Python and migrated a large number of test cases from other popular databases. We implemented the program again with [sqllogictest-rs](https://github.com/risinglightdb/sqllogictest-rs) for Rust in recent days.